core/consensus/grandpa/impl/grandpa_impl.hpp

```diff
@@ -53,58 +77,176 @@ namespace kagome::consensus::grandpa {
                 std::shared_ptr<network::PeerManager> peer_manager,
                 std::shared_ptr<blockchain::BlockTree> block_tree);
 
-    /** @see AppStateManager::takeControl */
+    /**
```

Comment:
what are other options? I will try to make sure that this docs are up to date as part of code review

Reply:
The other option is not writing the internals of this method and just write that it executes logic required to prepare GrandpaImpl for running. If anybody needs more specific details, it would in any case be better to just go and read the function body.